python/ray/tune/examples/benchmarks/FCNet_Trainable.py

```diff
@@ -0,0 +1,89 @@
+""" Trainable using FCNet Benchmark from Tabular Benchmarks for Hyperparameter
+    Optimization and Neural Architecture Search """
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import ray
+from ray.tune import Trainable
+import tarfile
+import urllib
+import time
+import json
+from nas_benchmarks.tabular_benchmarks import FCNetProteinStructureBenchmark, \
+    FCNetSliceLocalizationBenchmark, FCNetNavalPropulsionBenchmark, \
+    FCNetParkinsonsTelemonitoringBenchmark
+
+
+class FCNetTrainable(Trainable):
+    def _setup(self, config):
+
+        # download dataset
+        file_tmp = urllib.urlretrieve(
+            "http://ml4aad.org/wp-content/uploads/2019/01" +
+            "/fcnet_tabular_benchmarks.tar.gz",
+            "./fcnet_tabular_benchmarks")[0]
+        base_name = os.path.basename(
+            "http://ml4aad.org/wp-content/uploads/2019/01" +
+            "fcnet_tabular_benchmarks.tar.gz")
+
+        file_name, file_extension = os.path.splitext(base_name)
+        tar = tarfile.open(file_tmp)
+        tar.extractall(file_name)
+
+        self._global_start = config.get("start", time.time())
+        self._trial_start = time.time()
+        self.config = config
+        cwd = os.getcwd()
+        if ray.worker._mode() == ray.worker.LOCAL_MODE:
+            os.chdir(cwd)
+        self.iteration = 0
+
+    def _train(self):
+        acc, time = self.net.objective_function(self.config, self.iteration)
+        self.iteration += 1
+        return {"validation_accuracy": acc, "runtime": time}
+
+    def _save(self, checkpoint_dir):
```

Comment:
I think you don't need this since `self.iteration` is auto-tracked - this should probably just be 
```python
def _save(self, checkpoint_dir):
    return checkpoint_dir

def _restore(self, checkpoint_dir):
    pass
```